src/TuDa.CIMS.Web/Components/Pages/InvoicePage.razor.cs

```diff
@@ -0,0 +1,75 @@
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using MudBlazor;
+using TuDa.CIMS.Shared.Entities;
+using TuDa.CIMS.Shared.Models;
+using TuDa.CIMS.Web.Services;
+
+namespace TuDa.CIMS.Web.Components.Pages;
+
+public partial class InvoicePage
+{
+    /// <summary>
+    /// Parameter for the working group.
+    /// </summary>
+    [Parameter]
+    public static WorkingGroup? WorkingGroup { get; set; }
+
+    // Error when WorkingGroup is not static occurs.
+    /// <summary>
+    /// Contains the title and the name of the buyer.
+    /// </summary>
+    private readonly string BuyerString = WorkingGroup.Professor.Title + " " + WorkingGroup.Professor.Name;
```

Comment:
🔧 This needs to be a property. If not, the evaluation is done only once
```suggestion
    /// </summary>
    private string BuyerString => WorkingGroup.Professor.Title + " " + WorkingGroup.Professor.Name;
```